index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine">
+    <title>Medicore </title>
+    <link href="./styles.css" rel="stylesheet">
+</head>
+<body>
+    <div class="header">
+        <div class="menu-bar">
+            <div class="menu-bar__logo menu-bar__logo--text-big">
+                <div class="menu-bar__link">Medical Core</div>
+            </div>
+            <div class="menu-bar__links menu-bar__links--text-muted">
+                <div class="menu-bar__link">Home</div>
+                <div class="menu-bar__link">Product</div>
+                <div class="menu-bar__link">Pricing</div>
+                <div class="menu-bar__link">Contact</div>
+            </div>
+            <div class="menu-bar__section">
+                <button class="button button--no-background">Login</button>
+                <button class="button button--uppercase button--background">join us <img src="./images/Vector.png"> </button>
+            </div>
+        </div>
+        <div class="welcome">
+            <div class="welcome__text">
+                <label class="welcome__desc welcome__text--primary welcome__text--big-in-sm">Welcome</label>
+                <label class="welcome__desc welcome__text--big">A Great Place to Receive Care</label>
```

Comment:
Same issue here.

Reply:
i changed it to ablock because i will re-use it in the footer area